packages/plugin-core/src/utils/md.ts

```diff
@@ -237,11 +235,22 @@ export type getReferenceAtPositionResp = {
   refText: string;
 };
 
-export const getReferenceAtPosition = (
-  document: vscode.TextDocument,
-  position: vscode.Position,
-  opts?: { partial?: boolean; allowInCodeBlocks: boolean }
-): getReferenceAtPositionResp | null => {
+export const getReferenceAtPosition = async ({
```

Comment:
could we use a named function here like `export async function getReferenceAtPosition` so the name will show up in backtraces?

Reply:
updated. and good point. can you write an async about this and update [[Best Practices|dendron://dendron.docs/dev.process.code.best-practices]]?